public/examples/additional_py.py

```diff
@@ -0,0 +1 @@
+print("Imported additionalPy.py")
```

Comment:
added an example for importing a .py file from an additional file (which no longer cares about escaping ")